include/network/uri/uri_builder.hpp

```diff
@@ -182,13 +182,29 @@ class uri_builder {
   uri_builder &clear_path();
 
   /**
+   * \deprecated Please use append_query_parameter
+   * \warning This function's behaviour has changed and percent encoding
+   *          of the '=' character is not ignored.
    * \brief Adds a new query to the uri_builder.
    * \param query The query.
    * \returns \c *this
+   * \sa append_query_parameter
    */
   template <typename Source>
   uri_builder &append_query(const Source &query) {
-    append_query(detail::translate(query));
+    return append_query_parameter(query);
+  }
+
+  /**
+   * \brief Adds a new query value to the uri_builder. The '='
+   *        symbol is percent encoded.
+   * \param parameter The query parameter.
+   * \returns \c *this
+   * \sa append_query_key_value_pair
+   */
+  template <typename Source>
+  uri_builder &append_query_parameter(const Source &parameter) {
```

Comment:
I would like to use consistent naming here, similar to cppnetlib's `header` type. A query parameter is either a single name or a name/value pair. So the signature would be:

`append_query_parameter(const Source &name)`